apps/nextjs/src/lib/feature-flags/bootstrap.ts

```diff
@@ -32,15 +32,21 @@ function getDistinctIdFromCookie(headers: ReadonlyHeaders) {
 }
 
 export async function getBootstrappedFeatures(headers: ReadonlyHeaders) {
-  const { userId } = auth();
+  const { userId, sessionClaims } = auth();
 
   const distinctId = userId ?? getDistinctIdFromCookie(headers) ?? "0";
   log.info("Evaluating feature flags for", distinctId);
+
+  const personProperties = sessionClaims?.labs?.featureFlagGroup
+    ? { featureFlagGroup: sessionClaims.labs.featureFlagGroup }
+    : undefined;
+
   const features = await posthogAiBetaServerClient.getAllFlags(distinctId, {
-    // Only bootstrap flags which don't depend on user properties
-    // These are typically flags representing new features
+    // Only bootstrap flags which don't depend on PII
     onlyEvaluateLocally: true,
+    personProperties,
   });
+
   log.info("Bootstrapping feature flags", features);
```

Comment:
Worth logging some of the new properties here?

Reply:
Nice idea. Will do